nautobot/core/templates/inc/custom_fields/panel_data.html

```diff
@@ -3,37 +3,58 @@
-                        {% elif field.type == 'multi-select' and value %}
-                            {{ value|join:", " }}
-                        {% elif field.type == 'json' and value is not None %}
-                            <p>
-                                <button class="btn btn-xs btn-primary" type="button" data-toggle="collapse"
-                                        data-target="#cf_{{ field.name | slugify }}" aria-expanded="false"
-                                        aria-controls="cf_{{ field.name | slugify }}">
-                                    Show/Hide
-                                </button>
-                            </p>
-                            <pre class="collapse" id="cf_{{ field.name | slugify }}"><code>{{ value|render_json }}</code></pre>
-                        {% elif value or value == 0 %}
-                            {{ value }}
-                        {% elif field.required %}
-                            <span class="text-warning">Not defined</span>
-                        {% else %}
-                            <span class="text-muted">&mdash;</span>
-                        {% endif %}
-                    </td>
-                </tr>
-            {% endfor %}
+        <table id="accordion" class="table table-hover panel-body attr-table">
+            {% for grouping, fields in custom_fields.items %}
+            {% with forloop.counter0 as count %}
+                {% if grouping != "" %}
+                    <tr>
+                        <!-- #TODO
+                        Here we are using forloop.counter0 to guarantee the unqiueness of `data-target` attribute.
+                        This feature on the joblist page is using {{ grouping | slugify }} it should be changed as it breaks `accordion-toggle`
+                        when two distinct groups have the same slugified name. e.g. Group 2, group-2 -->
+                        <th>
+                            <button type="button" class="btn-link accordion-toggle mdi mdi-chevron-down"
+                            name="grouping.{{ grouping }}" data-toggle="collapse"
+                            data-target=".collapseme-{{ count }}">
+                                {{ grouping }}
+                            </button>
+                        </th>
+                        <td></td>
+                    </tr>
+                {% endif %}
+                    {% for field, value in fields %}
+                        <tr class="collapseme-{{ count }} collapse in" data-parent="#accordion">
+                            <td><span title="{{ field.description }}">{{ field }}</span></td>
+                            <td>
+                                {% if field.type == 'boolean' %}
+                                    {{ value | render_boolean }}
+                                {% elif field.type == 'url' and value %}
+                                    <a href="{{ value }}">{{ value|truncatechars:70 }}</a>
```

Comment:
I know this is present in the base template too but it does seem odd to me that we are truncating the displayed URL here.

Reply:
<img width="1792" alt="Screen Shot 2022-08-26 at 1 00 42 PM" src="https://user-images.githubusercontent.com/46973263/186955686-01b0c8fb-55d1-44d3-a3db-cca60a2ed97a.png">
I guess this is the reason lol